### PR TITLE
[AD-473] Publish jar file to Maven Central repository

### PR DESCRIPTION
**Pull request by affonsov**
### Summary

Publish jar file to Maven Central repository

### Related Issue

https://bitquill.atlassian.net/browse/AD-473

### Additional Reviewers
@affonsoBQ
@alinaliBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (affonsov) [inline pom.xml:1]: I'm not sure if needs license header

Comment (affonsov) [inline pom.xml:1]: I'm not sure if needs license header

Comment (alexey-temnikov) [reply pom.xml:1]: pom file doesn't need it.

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%m%n
+log4j.appender.console.target=System.out
+log4j.appender.console.immediateFlush=true
+log4j.appender.console.encoding=UTF-8
+log4j.appender.console.Threshold=${documentdb.jdbc.log.console.threshold}
+
+log4j.rootLogger=${documentdb.jdbc.log.level}, rollingFile, console
+log4j.logger.performance=INFO, rollingFile
```
Comment (alexey-temnikov) [inline src/main/resources/log4j.properties]: nit. new line

**Review by alexey-temnikov — APPROVED** [review]
LGTM overall.

**Pull request merged**